h2o-docs/src/front/index.html

```diff
@@ -1,970 +1,207 @@
-            <tbody class="lt-gry-bg dltile">
-            <tr>
-              <td style="padding-bottom: 3pt; padding-top: 3pt; padding-right: 0pt; padding-left: 2pt;">Enterprise Steam Installation Guide</td>
-              <td style="padding-bottom: 3pt; padding-top: 3pt; "align="right"><a href="http://docs.h2o.ai/enterprise-steam/latest-stable/docs/install-docs/_build/html/index.html">HTML</a></td>
-              <td style="padding-bottom: 3pt; padding-top: 3pt; "align="right"><a href="http://docs.h2o.ai/enterprise-steam/latest-stable/docs/install-docs/esteaminstallandsetup.pdf">PDF</a></td>
-            </tr>
-            <tr>
-              <td style="padding-bottom: 3pt; padding-top: 3pt; padding-right: 0pt; padding-left: 2pt;">Enterprise Steam User Guide</td>
-              <td style="padding-bottom: 3pt; padding-top: 3pt;" align="right"><a href="http://docs.h2o.ai/enterprise-steam/latest-stable/docs/user-docs/_build/html/index.html">HTML</a></td>
-              <td style="padding-bottom: 3pt; padding-top: 3pt;" align="right"><a href="http://docs.h2o.ai/enterprise-steam/latest-stable/docs/user-docs/esteamuserguide.pdf">PDF</a></td>
-            </tr> 
-          </tbody>
-          </table>
-          </div>
-          <hr>
-          <div class="primary-commercial txt-cntr" style="display: block; width: 100%; margin-bottom:10px;" href="mailto:sales@h2o.ai">
-            <a href="https://www.h2o.ai/download-enterprise-steam/" style="color: #ffdf00; font-size: 1.1em;">Download Enterprise Steam</a>
-          </div>
-        </div>
-      </div>
 
-      <div class="flex-vert col-3">
-        <div class="well well-sm">
-          <h5>Steam</h5>
-          <div class="lt-gry-bg1 dltile" style="display: flex !important; display: -webkit-flex !important; flex-direction: column; padding: 6px;">
-            <a href="http://www.h2o.ai/steam/">What is Steam?</a>
-            <a href="http://docs.h2o.ai/steam/latest-stable/index.html">Steam User Guide</a>
-            <a href="https://github.com/h2oai/steam/blob/master/ReleaseNotes.md">Recent Changes</a>
-            <a style="line-height: 1.3em"; href="https://github.com/h2oai/steam/blob/master/LICENSE">Open Source License (AGPL)</a>
+<!-- h2o-3 section -->
+
+  <section class="download-products bg-lighter-grey download-with-nav" id="h2o">
+    <div class="container">
+      <div class="column-wrapper">
+        <div class="col-two left text-left">
+          <h4 class="underline-yellow wow wefadeInUp">H2O-3</h4>
+          <h6 class="wow wefadeInUp">The H2O open source platform works with R, Python, an Scala on Hadoop/Yarn, Spark, or your laptop</h6>
+        </div>
+
+        <div class="col-two right">
+
+          <div class="bottom-wrapper wow wefadeInUp" style="padding-top: 30px;">
+              <h6>End User Documentation</h6>
+              <div class="col-one" style="padding-top: 15px;">
+                <p><a href="/h2o/latest-stable/h2o-docs/index.html">H2O User Guide</a></p>
```

Comment:
I don't think the relative links are correct. I would imagine you have to drop the `/h2o/latest-stable` prefix on all of them since the index.html would in this kind of directory structure:
https://s3.console.aws.amazon.com/s3/buckets/h2o-release/h2o/rel-yule/1/docs-website/?region=us-east-1&tab=overview#

Reply:
Oh, you're correct. I updated these so that they're relative to the docs-website folder.